roles/control_node/defaults/main.yml

```diff
@@ -9,3 +9,16 @@ network_ee: "registry.redhat.io/ansible-automation-platform-20-early-access/ee-s
 rhel_90_ee: "registry.redhat.io/ansible-automation-platform-20-early-access/ee-supported-rhel8:2.0.0"
 windows_ee: "quay.io/acme_corp/windows_ee:latest"
 smart_mgmt_ee: "quay.io/s4v0/ee-automated-smart-mgmt-29:1.0.4"
+
+
+# EE registry name used in installer and EE controller credential
+ee_registry_name: registry.redhat.io
+
+# EEs to download for awx user
+ee_images:
+  - "{{ ee_registry_name }}/ansible-automation-platform-20-early-access/ee-supported-rhel8:2.0.1"
```

Comment:
why not point to latest?  are we worried they will be buggy?

Reply:
I remember we had issues with using `:latest` tag for EEs. It always tried to fetch the latest version before it executes the job template. Is that still accurate? If not, I'll change it to `:latest`